src/components/UI/PasswordInput.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="wrapper">
+    <label for="password" class="password-label">Password</label>
+    <input
+      class="password-input"
+      required
+      :type="inputType"
+      placeholder="Enter your password"
+      :value="value"
+      name="password"
+      @input="$emit('input', $event.target.value)"
+      @blur="$emit('blurTouched')"
+    />
+    <button class="password-eye" @click.prevent="toggleViewPassword">
```

Comment:
you needed to add `.prevent` to not submit a form with this button, right? This is because if the attribute `type` is not specified for buttons associated with a `<form>` it works like `<button type="submit">`

the better idea would be to just add `type="button"` 😉